src/com/com_interface.hpp

```diff
@@ -12,58 +12,58 @@
 
 namespace com {
 
+/** 
+ * @brief This is a purely virtual class to act as
+ * a facade for the device specific implementation.
+ * 
+ */
+class ComInterface {
+ public:
   /** 
-   * @brief This is a purely virtual class to act as
-   * a facade for the device specific implementation.
+   * @brief Default constructor may not be used.
    * 
    */
-  class ComInterface {
-    public:
-      /** 
-       * @brief Default constructor may not be used.
-       * 
-       */
-      ComInterface() = delete;
+  ComInterface() = delete;
 
-      /**
-       * @brief Construct a new Com Interface object
-       * 
-       * @param msg_buf Pointer to an instance of a com message buffer object.
-       */
-      explicit ComInterface(std::unique_ptr<com::ComMessageBuffer> msg_buf) : msg_buffer_(std::move(msg_buf)){};
+  /**
+   * @brief Construct a new Com Interface object
+   * 
+   * @param msg_buf Pointer to an instance of a com message buffer object.
+   */
+  explicit ComInterface(std::unique_ptr<com::ComMessageBuffer> msg_buf) : msg_buffer_(std::move(msg_buf)){};
+
+  /**
+   * @brief Virtual destructor. 
+   *
```

Comment:
Just for the sake of interest:
Is there a reason for this blank line after `@brief`? 
Of course, we don't have to change that.

Reply:
I just re-formatted the comments. Please ask @c0nwild about this. Maybe it is created by doxygen documentation generator?

Reply:
This is what I suspect too. My guess is, doc. generator always puts an empty line there in case there are more comments to be generated. Have to check if this behavior is configurable.